frontend/scripts/update-reflame-environment.mjs

```diff
@@ -4,30 +4,37 @@ import * as fs from 'node:fs'
 const configPath = '../.reflame.config.jsonc'
 const configText = fs.readFileSync(configPath, 'utf8')
 
+const name = process.argv[2]
+
+if (!name) {
+	throw new Error('missing environment name')
+}
+
 const result = jsoncParser.applyEdits(
 	configText,
 	jsoncParser.modify(
 		configText,
-		['environment'],
+		['environments', name],
 		{
-			REACT_APP_FIREBASE_CONFIG_OBJECT: process.env
-				.REACT_APP_FIREBASE_CONFIG_OBJECT ?? {
-				type: 'expression',
-				value: "JSON.stringify({apiKey: 'AIzaSyD7g86A3EzEKmoE7aZ04Re3HZ0B4bWlL68',authDomain: 'auth.highlight.run',databaseURL: 'https://highlight-f5c5b.firebaseio.com',projectId: 'highlight-f5c5b',storageBucket: 'highlight-f5c5b.appspot.com',messagingSenderId: '263184175068',appId: '1:263184175068:web:f8190c20320087d1c6c919',})",
-			},
-			REACT_APP_COMMIT_SHA: process.env.REACT_APP_COMMIT_SHA ?? {
+			REACT_APP_FIREBASE_CONFIG_OBJECT:
+				process.env.REACT_APP_FIREBASE_CONFIG_OBJECT?.replaceAll(
+					'\n',
+					'',
+				) ??
+				"{apiKey: 'AIzaSyD7g86A3EzEKmoE7aZ04Re3HZ0B4bWlL68',authDomain: 'auth.highlight.run',databaseURL: 'https://highlight-f5c5b.firebaseio.com',projectId: 'highlight-f5c5b',storageBucket: 'highlight-f5c5b.appspot.com',messagingSenderId: '263184175068',appId: '1:263184175068:web:f8190c20320087d1c6c919',}",
+			REACT_APP_COMMIT_SHA: {
 				type: 'expression',
 				value: 'Reflame.gitCommitSha',
 			},
 			REACT_APP_PRIVATE_GRAPH_URI:
 				process.env.REACT_APP_PRIVATE_GRAPH_URI ??
 				'https://pri.highlight.run',
 			REACT_APP_ONPREM: process.env.REACT_APP_ONPREM ?? false,
-			REACT_APP_FRONTEND_URI: process.env.REACT_APP_FRONTEND_URI ?? {
+			REACT_APP_FRONTEND_URI: {
 				type: 'expression',
 				value: 'window.location.origin',
 			},
```

Comment:
interesting... yeah - i don't think this should ever differ from `window.location.origin`. i can clean this up in your fork

Reply:
This can probably wait, doesn't hurt to keep this around until you have the time to refactor and make sure all the different dev flows still work as expected.